src/js/webSerial.js

```diff
@@ -40,10 +40,7 @@ class WebSerial extends EventTarget {
 
     async connect(options) {
         this.openRequested = true;
-        this.port = await navigator.serial.requestPort({ filters: [
-            // TODO: what other vendors does betaflight recognize?
-            { usbVendorId: 1155 },
-        ]});
+        this.port = await navigator.serial.requestPort();
```

Comment:
https://github.com/betaflight/betaflight-configurator/blob/55fabf2c49737bc54099407f3fe56913bfa6d681/src/js/serial.js#L25-L32

Reply:
We use the filtering to only show and use relevant devices